### PR TITLE
SAK-49678 WC sakai-editor not loading in conversations

### PR DESCRIPTION
**Pull request by ern**
https://sakaiproject.atlassian.net/browse/SAK-49678

### CODE REVIEW

**Review by adrianfish — APPROVED** [review]
Great, thanks.

File: webcomponents/tool/src/main/frontend/packages/sakai-conversations/src/SakaiAddTopic.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { html } from "lit";
 import { SakaiElement } from "@sakai-ui/sakai-element";
-import "@sakai-ui/sakai-editor";
+import "@sakai-ui/sakai-editor/sakai-editor.js";
 import "@sakai-ui/sakai-icon";
 import "@sakai-ui/sakai-date-picker";
```
Comment (ottenhoff) [inline webcomponents/tool/src/main/frontend/packages/sakai-conversations/src/SakaiAddTopic.js:5]: why does sakai-editor need reference to a specific JS file and sakai-date-picker does not?

Comment (adrianfish) [reply webcomponents/tool/src/main/frontend/packages/sakai-conversations/src/SakaiAddTopic.js:5]: Might be the exports in package.json

Comment (ottenhoff) [reply webcomponents/tool/src/main/frontend/packages/sakai-conversations/src/SakaiAddTopic.js:5]: > Might be the exports in package.json

should be fixed there instead? 

obv still trying to learn here so you're not the only one expected to maintain sakai web components

Comment (adrianfish) [reply webcomponents/tool/src/main/frontend/packages/sakai-conversations/src/SakaiAddTopic.js:5]: I understand. It's good to see people getting stuck in. I'll have a look at some of these and try and work out what's going on.

**Pull request merged**